website/blog/2022-05-03-making-dbt-cloud-api-calls-using-dbt-cloud-cli.md

```diff
@@ -0,0 +1,327 @@
+---
+title: "Making dbt Cloud API calls using dbt-cloud-cli"
+description: "Simo Tumelius shares how to use his dbt-cloud-cli to makae dbt Cloud API calls."
```

Comment:
Wonder if we can make this a little more descriptive / engaging

Reply:
added some descriptors there. Lmk what you think!